mws/mwsauth/fixtures/mwsuser_admin_users.yaml

```diff
@@ -0,0 +1,12 @@
+- model: mwsauth.MWSUser
+  fields: { uid: 178460, user: amc203 }
+- model: mwsauth.MWSUser
+  fields: { uid: 53956, user: rjw53 }
```

Comment:
this user should be rjw57 with the associated uid

Reply:
Thanks Peter now corrected.